CSharp/AccountManagement/Program.cs

```diff
@@ -1,334 +1,349 @@
+                Console.WriteLine("Your subscription can create {0} account(s) in the {1} region.", quotaResponse.AccountQuota, location);
+                Console.WriteLine();
+
+                // Create account
+                string accountName = PromptUserForAccountName();
+                Console.WriteLine("Creating account {0}...", accountName);
+                await batchManagementClient.Account.CreateAsync(ResourceGroupName, accountName, new BatchAccountCreateParameters() { Location = location });
+                Console.WriteLine("Account {0} created", accountName);
+                Console.WriteLine();
+
+                // Get account
+                Console.WriteLine("Getting account {0}...", accountName);
+                AccountResource account = await batchManagementClient.Account.GetAsync(ResourceGroupName, accountName);
+                Console.WriteLine("Got account {0}:", account.Name);
+                Console.WriteLine("  Account location: {0}", account.Location);
+                Console.WriteLine("  Account resource type: {0}", account.Type);
+                Console.WriteLine("  Account id: {0}", account.Id);
+                Console.WriteLine();
+
+                // Print account quotas
+                Console.WriteLine("Quotas for account {0}:", account.Name);
+                Console.WriteLine("  Core quota: {0}", account.CoreQuota);
+                Console.WriteLine("  Pool quota: {0}", account.PoolQuota);
+                Console.WriteLine("  Active job and job schedule quota: {0}", account.ActiveJobAndJobScheduleQuota);
+                Console.WriteLine();
+
+                // Get account keys
+                Console.WriteLine("Getting account keys of account {0}...", account.Name);
+                BatchAccountListKeyResult accountKeys = await batchManagementClient.Account.ListKeysAsync(ResourceGroupName, account.Name);
+                Console.WriteLine("  Primary key of account {0}:   {1}", account.Name, accountKeys.Primary);
+                Console.WriteLine("  Secondary key of account {0}: {1}", account.Name, accountKeys.Secondary);
+                Console.WriteLine();
+
+                // Regenerate primary account key
+                Console.WriteLine("Regenerating the primary key of account {0}...", account.Name);
+                BatchAccountRegenerateKeyResult newKeys = await batchManagementClient.Account.RegenerateKeyAsync(
+                    ResourceGroupName, account.Name,
+                    new BatchAccountRegenerateKeyParameters() { KeyName = AccountKeyType.Primary });
+                Console.WriteLine("  New primary key of account {0}: {1}", account.Name, newKeys.Primary);
+                Console.WriteLine("  Secondary key of account {0}:   {1}", account.Name, newKeys.Secondary);
+                Console.WriteLine();
+
+                // Print subscription quota information
```

Comment:
This comment seems inaccurate.  You're printing the total number of accounts in the subscription here.